web/styles/utils.sass

```diff
@@ -160,3 +160,13 @@
 
 .error-border
   border-color: $error !important
+
+.active-tab
+  border-bottom: 4px solid $primary
+
+  @media (hover:hover)
+    &:hover
+      border-bottom-color: $secondary-highlight
+
+.inactive-tab
+  border-bottom: 4px solid white
```

Comment:
I don't like this approach, let me know if you have a better solution. Basically the text is vertically centered, and the text of the active link has less vertical space because of the bottom border. So I just added the border to all links, and set the colour to white (bg is white too) if they're inactive to make them invisible 🤷‍♂️

Reply:
I think this approach is fine, but maybe we can also combine it into a single `.tab` class with the `inactive-tab` being the default state?

Ex:
```
.tab
  border-bottom: 4px solid white // inactive state by default
  &.active
    border-bottom: 4px solid $primary
  @media (hover:hover)
    &:hover
      border-bottom-color: $secondary-highlight
```